app/global-error.tsx

```
@@ -0,0 +1,180 @@
+                  <br />
+                  <strong>Error ID:</strong> {error.digest}
+                </>
+              )}
+            </div>
+          )}
+          
+          <button 
+            className="button" 
+            onClick={() => window.location.href = '/'}
+          >
+            Return Home
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)